addon/adapters/github.js

```diff
@@ -19,7 +19,8 @@ export default RESTAdapter.extend({
   }),
 
   pathForType(type) {
-    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')));
+    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')))
+      .replace('organizations', 'orgs');
```

Comment:
What you could do is generalize this...have each adapter define two properties (representing `organizations` & `orgs`, or their equivalent). Then the base adapter can `get` those properties to do the replace, if they have been defined.